src/viser/client/src/ControlPanel/MultiSlider.styles.tsx

```diff
@@ -0,0 +1,243 @@
+import { createStyles, rem } from '@mantine/styles';
+import { MantineColor, getSize, MantineNumberSize } from '@mantine/styles';
+
+
+export const sizes = {
+  xs: rem(4),
+  sm: rem(6),
+  md: rem(8),
+  lg: rem(10),
+  xl: rem(12),
+};
+
+export const useSliderRootStyles = createStyles((theme) => ({
+  root: {
+    ...theme.fn.fontStyles(),
+    WebkitTapHighlightColor: 'transparent',
+    outline: 0,
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    touchAction: 'none',
+    position: 'relative',
+  },
+}));
+
+
+interface ThumbStyles {
+  color: MantineColor;
+  disabled: boolean;
+  thumbSize: number | string;
+}
+
+export const useThumbStyles = createStyles((theme, { color, disabled, thumbSize }: ThumbStyles, { size }) => ({
+  label: {
+    position: 'absolute',
+    top: rem(-36),
+    backgroundColor: theme.colorScheme === 'dark' ? theme.colors.dark[4] : theme.colors.gray[9],
+    fontSize: theme.fontSizes.xs,
+    color: theme.white,
+    padding: `calc(${theme.spacing.xs} / 2)`,
+    borderRadius: theme.radius.sm,
+    whiteSpace: 'nowrap',
+    pointerEvents: 'none',
+    userSelect: 'none',
+    touchAction: 'none',
+  },
+
+  thumb: {
+    ...theme.fn.focusStyles(),
+    boxSizing: 'border-box',
+    position: 'absolute',
+    display: disabled ? 'none' : 'flex',
```

Comment:
it would be nice if thumbs didn't disappear entirely when the input is disabled, since it makes the "value" of the input invisible to the user

maybe we could make the thumbs grayscale or something? or set `cursor: not-allowed;` to the CSS (although this might be ugly)

Reply:
New disabled state:
![image](https://github.com/nerfstudio-project/viser/assets/4222114/7ddbf898-26b3-448d-9d27-a0e000a547ca)